node_modules/oae-core/preferences/preferences.html

```diff
@@ -111,7 +111,7 @@ <h3 id="preferences-modal-title">__MSG__MY_PREFERENCES__</h3>
                             <i class="fa fa-check hide"></i>
                             <div class="well oae-large-options">
                                 <input type="radio" id="preferences-email-immediate" value="immediate" name="preferences-email-group" class="pull-left" tabindex="0" {if oae.data.me.emailPreference === 'immediate'} checked{/if}/>
-                                <i class="fa fa-bolt large text-center {if oae.data.me.emailPreference === 'immediate'} selected{/if}"></i>
```

Comment:
It looks like the `selected` class is also being applied through js in the following cases:

```
node_modules/oae-admin/manageuser/js/manageuser.js:                $('.oae-large-options i', $optionsContainer).removeClass('selected');
node_modules/oae-admin/manageuser/js/manageuser.js:                $(this).closest('label').find('.oae-large-options i').addClass('selected');
node_modules/oae-core/preferences/js/preferences.js:                $('.oae-large-options i', $rootel).removeClass('selected');
node_modules/oae-core/preferences/js/preferences.js:                $(this).parents('label').find('.oae-large-options i').addClass('selected');
```